youtube-channels.csv

```diff
@@ -425,3 +425,13 @@ UCKvCGuur4JFEgScY-81Ph0Q, I AM ME, https://www.youtube.com/channel/UCKvCGuur4JFE
 UCHQs2gBIRQ0oLRHhSV5Jrig, House Nest, https://www.youtube.com/channel/UCHQs2gBIRQ0oLRHhSV5Jrig
 UCCfmUX1OoQVmqalSft-O5BA, Sieben, https://www.youtube.com/channel/UCCfmUX1OoQVmqalSft-O5BA
 UCzxogHW8GLn5W5AYbPmQJOg, Brutalist Empire, https://www.youtube.com/channel/UCzxogHW8GLn5W5AYbPmQJOg
+UCPDILkId0HJTU9A82ItZn2A, X-Dream, https://www.youtube.com/channel/UCPDILkId0HJTU9A82ItZn2A
+UCDo_VHBvW6g6esvbRLyqr8A, DEE BDUBZ, https://www.youtube.com/channel/UCDo_VHBvW6g6esvbRLyqr8A
+UCgZn1e9AK2GMFFL5pqxqPYg, AlexDaftPunk91, https://www.youtube.com/channel/UCgZn1e9AK2GMFFL5pqxqPYg
+UC3REe_sF54K57o_PcC_Pn_A, Aleix Dorian, https://www.youtube.com/channel/UC3REe_sF54K57o_PcC_Pn_A
+UCh780tXKV4HUqrIHiMz37Pw, OLDSKOOL 88 94, https://www.youtube.com/channel/UCh780tXKV4HUqrIHiMz37Pw
+UCygzjdPR6EZ7luNBHM5ksjA, MRDanceHouseRare, https://www.youtube.com/channel/UCygzjdPR6EZ7luNBHM5ksjA
+UCB6WUoQ-2im9Xhg9-24NtMA, ILoveHouseMusicTV, https://www.youtube.com/channel/UCB6WUoQ-2im9Xhg9-24NtMA
+UC5WsgjNrrgyNQnD6ATxtWtg, DJ Migmac O Max, https://www.youtube.com/channel/UC5WsgjNrrgyNQnD6ATxtWtg
```

Comment:
Unfortunately, in this channel most tracks seem to be `Track - Artist` instead of `Artist - Track`. Currently this is not supported as it will look for Artists called `Track` and tracks called `Artist`. Please remove it for now

Reply:
I've added it to https://github.com/mirrorfm/data/issues/31